Runtime/editor/editor_mouse_tracking.cs

```diff
@@ -0,0 +1,189 @@
+using System.Collections.Generic;
+using Unity.UIWidgets.foundation;
+using Unity.UIWidgets.scheduler;
+using UnityEditor;
+
+namespace Unity.UIWidgets.gestures {
```

Comment:
I think it is necessary to surround all the implementations below with a #if UNITY_EDITOR/#endif pair. It may helps us to ensure that, all these logics will never be called out of Editor and also make it explicit that "All the stuffs here are for Editor-use only" to anyone who reads this